rust/chains/hyperlane-sealevel/src/utils.rs

```diff
@@ -77,3 +79,14 @@ pub async fn get_account_metas(
 
     Ok(account_metas)
 }
+
+pub async fn get_finalized_block_number(rpc_client: &RpcClientWithDebug) -> ChainResult<u32> {
+    let height = rpc_client
+        .get_block_height()
+        .await
+        .map_err(ChainCommunicationError::from_other)?
+        .try_into()
+        // FIXME solana block height is u64...
```

Comment:
https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/2610
Should tag this new issue

Reply:
added as the last item in the "drive-by changes" section of the description 👍